src/TLuaInterpreter.cpp

```diff
@@ -272,7 +272,8 @@ double TLuaInterpreter::getVerifiedDouble(lua_State* L, const char* functionName
     return lua_tonumber(L, pos);
 }
 // No documentation available in wiki - internal function
-// See also: verifyBoolean
+// Raises a Lua error in case of an API usage mistake
```

Comment:
Good idea, but rename these to type and value errors instead for differentiation